src/components/composite/oodGallerySlideshow.js

```diff
@@ -8,6 +8,7 @@ import 'slick-carousel/slick/slick.css';
 
 const oodGallerySlideshow = ({ blok }) => {
   const [slideshow, setSlideshow] = useState(null);
+  const [modalSlideshow, setModalSlideshow] = useState(null);
```

Comment:
@Lyafel This works, but is there a reason that you use useState instead of useRef to set the reference for the slideshow and modal slidershow?